Soccar [UI]/src/nl/soccar/ui/input/Keyboard.java

```diff
@@ -17,6 +17,7 @@
 
     // Stores the keys that are being pressed at any time.
     private static final List<KeyCode> PRESSED_KEYS;
+    private static final InputController controller = InputController.getInstance();

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "controller" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1068)